python-spec/src/somacore/collection.py

```diff
@@ -1,7 +1,11 @@
 import abc
 from typing import Any, MutableMapping, Optional, Sequence, Type, TypeVar, overload
 
+# ruff: noqa
 import pyarrow as pa
+
+# ruff: noqa
+import pyarrow_hotfix
```

Comment:
This only needs to be included once in `somacore/__init__.py`, since importing any somacore module necessarily entails importing that file. Putting the lint silence on the same line:

```python
import pyarrow_hotfix  # noqa: F401
```

will ensure minimal lint-disabling and messing around with the formatting.

Reply:
https://github.com/single-cell-data/SOMA/pull/182/commits/1c2b483ba4a0067bf83b5312612d96812b922d24